azkaban-common/src/test/java/azkaban/utils/TestUtils.java

```diff
@@ -65,4 +70,17 @@ public static UserManager createTestXmlUserManager() {
     final UserManager manager = new XmlUserManager(props);
     return manager;
   }
+
```

Comment:
Does it make sense to move these common test methods to the test module so that they can be used in all other modules more easily? e.g. the new az-core module can't depend on the common module.

Reply:
Yes, why not.. Maybe there could be a separate pull request after _this_ to move `TestUtils` and the new `LambdaMatcher` into `test/`?

Reply:
That would be good.